desktop_version/src/BinaryBlob.h

```diff
@@ -1,9 +1,29 @@
 #ifndef BINARYBLOB_H
 #define BINARYBLOB_H
 
+#include <vector>
+
 /* Laaaazyyyyyyy -flibit */
 // #define VVV_COMPILEMUSIC
 
+#define TRACK_NAMES \
+	X("data/music/0levelcomplete.ogg") \
```

Comment:
Can we rename these macros to indicate a set of actions is being done? I saw the `X()` action in BinaryBlob.cpp first and I had no idea what was going on... something like:

```
#define TRACK_ACTIONS \
    FOREACH_TRACK("etc")
```

Reply:
Done, rebased and force-pushed.

Reply:
Actually, I'll change `TRACK_NAMES` to `TRACK_ACTIONS` (missed it the first time I read your comment!)

Reply:
Nevermind actually, I don't think `TRACK_ACTIONS` is clearer than `TRACK_NAMES`.